gpflow/utilities/bijectors.py

```diff
@@ -5,17 +5,50 @@
 from .. import config
 from .utilities import to_default_float
 
+
 __all__ = ["positive", "triangular"]
 
 
-def positive(lower: Optional[float] = None):
-    lower_value = config.default_positive_minimum()
-    if lower_value is None:
-        return tfp.bijectors.Softplus()
+_POSITIVE_BIJECTOR_MAP = {
+    "exp": tfp.bijectors.Exp,
+    "softplus": tfp.bijectors.Softplus,
+}
+
+
+class Shift(tfp.bijectors.AffineScalar):
```

Comment:
This class should be removed, it will confuse people. The print should reflect existing TFP bijectors, not GPflow's aliases.

Reply:
Yes, this makes sense